File: Problems/Tic Tac Toe/java/player/Player.java
package player;

import shape.Shape;

public class Player {
    private String name;
    private Shape shape;

    public Player(String name, Shape shape) {
        this.name = name;
        this.shape = shape;
    }

    public String getName(){
        return name;
    }

    public Shape getShape() {
        return shape;
    }
}

File: Problems/Tic Tac Toe/Golang/player/player.go
package player

import "tictactoe/shape"

type Player struct {
	Name  string
	Shape shape.Shape
}

func NewPlayer(name string, shape shape.Shape) *Player {
	return &Player{
		Name:  name,
		Shape: shape,
	}
}

func (p *Player) GetName() string {
	return p.Name
}

func (p *Player) GetShape() shape.Shape {
	return p.Shape
}
